management/src/main/java/com/yubico/yubikit/management/DeviceInfo.java

```diff
@@ -155,13 +173,16 @@ static DeviceInfo parse(byte[] response, Version defaultVersion) throws BadRespo
             enabledCapabilities.put(Transport.NFC, readInt(data.get(TAG_NFC_ENABLED)));
         }
 
+        boolean fips = (formFactor.value & 0x80) == 0x80;
+        boolean sky = (formFactor.value & 0x40) == 0x40;
```

Comment:
This won't work as the FIPS and SKY bits have already been masked out by FormFactor.valueOf() higher up. You need to save the int value from the TLV and use that here instead of formFactor.value.